### PR TITLE
OSDOCS-3023:  Overview page for the book 'updating clusters'

### PR DESCRIPTION
**Pull request by xenolinux**
OCP version: 4.9+
QE: @jiajliu 
[Preview](https://deploy-preview-44010--osdocs.netlify.app/openshift-enterprise/latest/updating/index.html)
[Jira](https://issues.redhat.com/browse/OSDOCS-3023)

### CODE REVIEW

File: updating/index.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4. For more information, see xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[Understanding {product-title} updates].
+
+[id='updating-clusters-overview-understanding-container-platform-updates']
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
```
Comment (jiajliu) [inline updating/index.adoc:13]: The same issue with https://github.com/openshift/openshift-docs/pull/43645#discussion_r840192502

File: updating/index.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+* xref:../updating/updating-restricted-network-cluster.adoc#update-mirror-repository_updating-restricted-network-cluster[Mirroring the {product-title} image repository]
+* xref:../updating/updating-restricted-network-cluster.adoc#updating-restricted-network-image-signature-configmap[Creating the image signature config map]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-restricted_updating-restricted-network-cluster[Updating the restricted network cluster]
+* xref:../updating/updating-restricted-network-cluster.adoc#images-configuration-registry-mirror_updating-restricted-network-cluster[Configuring image registry repository mirroring]
+* xref:../updating/updating-restricted-network-cluster.adoc#generating-icsp-object-scoped-to-a-registry_updating-restricted-network-cluster[Widening the scope of the mirror image catalog to reduce the frequency of cluster node reboots]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-install[Installing the OpenShift Update Service Operator]
```
Comment (jiajliu) [inline updating/index.adoc:88]: The same issue with https://github.com/openshift/openshift-docs/pull/43645#discussion_r840199897

**Comment by jiajliu** [comment]
Furthermore, we still miss https://docs.openshift.com/container-platform/4.9/updating/updating-hardware-on-nodes-running-on-vsphere.html.

**Comment by xenolinux** [comment]
> Furthermore, we still miss https://docs.openshift.com/container-platform/4.9/updating/updating-hardware-on-nodes-running-on-vsphere.html.



> Furthermore, we still miss https://docs.openshift.com/container-platform/4.9/updating/updating-hardware-on-nodes-running-on-vsphere.html.

Addressed- added the part

File: updating/index.adoc (hunk 1 above)
Comment (xenolinux) [reply updating/index.adoc:13]: Addressed

Comment (xenolinux) [reply updating/index.adoc:13]: Addressed

File: updating/index.adoc (hunk 2 above)
Comment (xenolinux) [reply updating/index.adoc:88]: Addressed

**Comment by xenolinux** [comment]
@jiajliu I addressed all the comments, does this look good now?

**Comment by jiajliu** [comment]
lgtm

File: updating/index.adoc [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+[id="updating-clusters-overview-understanding-container-platform-updates"]
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
+
+["updating-clusters-overview-upgrade-channels-and-releases"]
+== Understanding upgrading channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following information:
```
Comment (nalhadef) [inline updating/index.adoc:17]: - Change "Upgrading  channels and releases" to "Upgrade channels and releases."
- Change the last sentence to: "For more information, see the following:"

File: updating/index.adoc [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+
+[id="updating-clusters-overview-understanding-container-platform-updates"]
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
+
+["updating-clusters-overview-upgrade-channels-and-releases"]
+== Understanding upgrading channels and releases
```
Comment (nalhadef) [inline updating/index.adoc:16]: Change "Upgrading  channels and releases" to "Upgrade channels and releases."

File: updating/index.adoc [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following information:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding Fast and stable channel use and strategies]
```
Comment (nalhadef) [inline updating/index.adoc:20]: In the link text, change "Fast" to "fast."

File: updating/index.adoc [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp_update-using-custom-machine-config-pools[Creating machine config pools to perform a canary rollout update]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-pause_update-using-custom-machine-config-pools[Pausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-update_update-using-custom-machine-config-pools[Performing the cluster update]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-unpause_update-using-custom-machine-config-pools[Unpausing the machine config pools]
+* xref:../updating/update-using-custom-machine-config-pools.adoc#update-using-custom-machine-config-pools-mcp-remove_update-using-custom-machine-config-pools[Moving a node to the original machine config pool]
```
Comment (nalhadef) [inline updating/index.adoc:58]: In the link text, change "config" to "configuration."

Also, go to the linked topic and change "config" to "configuration" in the titles and content.

File: updating/index.adoc [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-delete-service[Deleting an OpenShift Update Service application]
+* xref:../updating/updating-restricted-network-cluster.adoc#update-service-uninstall[Uninstalling the OpenShift Update Service Operator]
+
+["updating-clusters-overview-vsphere-updating-hardware"]
+== Updating hardware on nodes running on vSphere
+
+xref:../updating/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-hardware-on-nodes-running-on-vsphere[Updating hardware on vSphere]: You must ensure that your nodes running in vSphere are running on the hardware version supported by OpenShift Container Platform. Currently, hardware version 13 or later is supported for vSphere virtual machines in a cluster. For more information, see the following information:
```
Comment (jinyunma) [inline updating/index.adoc:86]: Suggest to copy the content of important warning statement in doc [Updating hardware on vSphere] to here and let user know whether they need to update hw version or not.

> Using hardware version 13 for your cluster nodes running on vSphere is now deprecated. This version is still fully supported, but support will be removed in a future version of OpenShift Container Platform. Hardware version 15 is now the default for vSphere virtual machines in OpenShift Container Platform.

File: updating/index.adoc (hunk 3 above)
Comment (xenolinux) [reply updating/index.adoc:17]: Addressed

File: updating/index.adoc (hunk 4 above)
Comment (xenolinux) [reply updating/index.adoc:16]: Addressed

File: updating/index.adoc (hunk 5 above)
Comment (xenolinux) [reply updating/index.adoc:20]: Addressed

File: updating/index.adoc (hunk 6 above)
Comment (xenolinux) [reply updating/index.adoc:58]: Addressed

File: updating/index.adoc (hunk 7 above)
Comment (xenolinux) [reply updating/index.adoc:86]: Addressed

File: updating/index.adoc [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+include::modules/common-attributes.adoc[]
+:context: updating-clusters-overview
+
+toc::[]
+
+You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+
+[id="updating-clusters-overview-understanding-container-platform-updates"]
+== Understanding OpenShift Container Platform updates
+xref:../updating/understanding-openshift-updates.adoc#understanding-openshift-updates[About the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc]: Addressed

[the remark this answers is not in the document]
Comment (xenolinux) [reply updating/index.adoc]: Addressed

File: updating/index.adoc (hunk 6 above)
Comment (xenolinux) [reply updating/index.adoc]: yes this PR will take care of it https://github.com/openshift/openshift-docs/pull/44734

**Pull request merged**